Source/Renderer/UniformState.js

```diff
@@ -261,7 +261,7 @@ define([
          * @memberof UniformState.prototype
          * @private
          */
-        inverseTranposeModel : {
+        inverseTransposeModel : {
```

Comment:
Good catch.  Can you please open a PR into master for this and the Model.js update?